react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -144,6 +155,7 @@ class OverflowMenu extends PureComponent<Props, State> {
                 <SecurityDialogButton { ...buttonProps } />
                 <RecordButton { ...buttonProps } />
                 <LiveStreamButton { ...buttonProps } />
+                {_salesforceEnabled && <LinkToSalesforceButton { ...buttonProps } />}
```

Comment:
can we move this check in the button code? I think there's a `visible` prop. Wdyt?

Reply:
good idea